Cargo.toml

```diff
@@ -49,6 +49,7 @@ risc0-r0vm = { version = "0.19.0-rc.1", default-features = false, path = "risc0/
 risc0-sys = { version = "0.19.0-rc.1", default-features = false, path = "risc0/sys" }
 risc0-zkp = { version = "0.19.0-rc.1", default-features = false, path = "risc0/zkp" }
 risc0-zkvm = { version = "0.19.0-rc.1", default-features = false, path = "risc0/zkvm" }
+risc0-zkvm-methods = { version = "0.19.0-rc.1", default-features = false, path = "risc0/zkvm/methods" }
```

Comment:
This command fails by having this be in the workspace:

```
$ cargo package -p risc0-zkvm
error: failed to prepare local package for uploading

Caused by:
  no matching package named `risc0-zkvm-methods` found
  location searched: registry `crates-io`
  required by package `risc0-zkvm v0.19.0-rc.1 ($SRC_ROOT/risc0/risc0/zkvm)`
```

Reply:
Oh interesting! I'm surprised this makes a difference. I've reverted this change in 616c42ca446e7f1243edb1fd18bcc033b609bcb8